framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -354,6 +354,39 @@ protected final QualifierDefaults createQualifierDefaults() {
     }
 
     /**
+     * Creates and returns a string containing the number of qualifiers and the
+     * canonical class names of each qualifier that has been added to this
+     * checker's supported qualifier set. The names are alphabetically sorted.
+     *
+     * @return a string containing the number of qualifiers and canonical names
+     *         of each qualifier.
+     */
+    private final String getSortedQualifierNames() {
```

Comment:
Why is this method private and not protected? Might errors in subclasses use this method?